test/unit/test_financial_functions.py

```diff
@@ -0,0 +1,70 @@
+import pytest
+
+# make sure we can find the app code
+import sys, os
+my_path = os.path.dirname(os.path.abspath(__file__))
+sys.path.insert(0, my_path + '/../../code/')
```

Comment:
We should add a TODO to not need to manually tweak the path. I'm guilty of this too.

Reply:
Yeah I was struggling with this a bit. What's the "right" way to do this? Any examples?